openvdb_houdini/SOP_NodeVDB.h

```diff
@@ -61,6 +61,22 @@ class OPENVDB_HOUDINI_API OpenVDBOpFactory: public houdini_utils::OpFactory
     /// Construct an OpFactory that on destruction registers a new OpenVDB operator type.
     OpenVDBOpFactory(const std::string& english, OP_Constructor, houdini_utils::ParmList&,
         OP_OperatorTable&, houdini_utils::OpFactory::OpFlavor = SOP);
+
+    /// Register the node
+    ~OpenVDBOpFactory();
```

Comment:
Unfortunately, `~OpFactory` is non-virtual; but since we don't delete `OpenVDBOpFactory` objects through base class pointers, this is probably OK for now.  Make a note, though, to make `~OpFactory` virtual in OpenVDB 7.

Reply:
That's true. I made a ticket to track this one - https://jira.aswf.io/browse/OVDB-102